TeacherCourse/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Comment:
You should put this line in a comment because you didn't use this import and the flake8 test would not pass because of it.